ios/RNStartTime.m

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+#import "RNStartTime.h"
+#import <sys/sysctl.h>
+#include <math.h>
+#import <QuartzCore/QuartzCore.h>
+
+@implementation RNStartTime
+
+RCT_EXPORT_MODULE()
+
+static CFTimeInterval processStartTime() {
+  size_t len = 4;
+  int mib[len];
+  struct kinfo_proc kp;
+  
+  sysctlnametomib("kern.proc.pid", mib, &len);
+  mib[3] = getpid();
+  len = sizeof(kp);
+  sysctl(mib, 4, &kp, &len, NULL, 0);
+  
+  struct timeval startTime = kp.kp_proc.p_un.__p_starttime;
+  return startTime.tv_sec + startTime.tv_usec / 1e6;
```

Comment:
I would appreciate some insight here

Reply:
Yes, I will actually document this bit.
Terry suggested me this library https://github.com/doomsower/react-native-startup-time
I didn't want to us it as a whole since it required calling a native module method to get the time it took to start, but it had implemented it using something noted in this article here https://medium.com/@michael.eisel/measuring-your-ios-apps-pre-main-time-in-the-wild-98197f3d95b4

I used the concept there to get the start timestamp of the app process